recipe/meta.yaml

```diff
@@ -7,46 +7,45 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meson_python-{{ version }}.tar.gz
```

Comment:
you can use name variable here instead of meson_python

Reply:
The name variable does not match "meson_python" and contains an underscore rather than a hyphen. This causes the URL to not be found.